### PR TITLE
fix(audio): Preload audio tracks

### PR DESCRIPTION
**Pull request by hugooliveirad**
Prevents audio tag to continuosly loading the audio files from the
network, what brings an awful experience for players over slow networks.

:metal:

### CODE REVIEW

File: src/assets/js/components/timer.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default class Timer extends Component {
   }
 
   onCountDownDone = (data) => {
-    matchActions.overtime();
+    //matchActions.overtime();
```
Comment (weslleyaraujo) [inline src/assets/js/components/timer.js]: a typo here 😛

Comment (rafaelrinaldi) [inline src/assets/js/components/timer.js]: @hugobessaa Ops?

File: src/assets/js/helpers/get-sound.js [hunk 2]
```diff
@@ -5,6 +5,11 @@ const sounds = {
   winner: '/dist/assets/sounds/winner.wav',
 }
 
+let soundsSrc = [];
+for (let src in sounds) {
+  soundsSrc.push(sounds[src]);
+}
+
```
Comment (weslleyaraujo) [inline src/assets/js/helpers/get-sound.js:11]: what you think of something like:

``` js
import { keys } from 'underscore';
const soundsSrc = keys(sounds).map(x => sound[src]);
```

P E R F E I T O

**Comment by hugooliveirad** [comment]
@rafaelrinaldi @weslleyaraujo should be better now

File: src/assets/js/components/audio.js [hunk 3]
```diff
@@ -25,7 +29,9 @@ export default class Audio extends Component {
   render = () => {
     return (
         <div>
-          <audio ref='player' src={this.props.sound}></audio>
+          {this.props.sounds.map((sound) =>
+            <audio preload="auto" ref={sound} src={sound} key={sound}></audio>
```
Comment (rafaelrinaldi) [inline src/assets/js/components/audio.js]: Double quotes :eyes:

Comment (hugooliveirad) [inline src/assets/js/components/audio.js]: where? :stuck_out_tongue_closed_eyes:

**Comment by weslleyaraujo** [comment]
thanks a lot!
that was something really necessary, and it will make a huge difference..

you rock dude 🚀

**Pull request merged**